astrocut/utils/utils.py

```diff
@@ -216,3 +217,7 @@ def get_fits(cutout_hdus, center_coord=None, output_path=None):
     return cutout_hdulist
 
 
+def _handle_verbose(verbose: bool):
+    """Set the log level according to the verbose parameter"""
+    level = logging.INFO if verbose else logging.WARNING
```

Comment:
I think we should rework the levels here.

verbose should turn on DEBUG level.

and the default should probably include INFO

so anything that was always printed to the screen should be INFO or above. Anything that was gated by `verbose` should be debug.

Reply:
Changed in newest commit!